geometry-in-ruby.gemspec

```diff
@@ -3,13 +3,13 @@ $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = "geometry-in-ruby"
-  s.version     = '0.0.6'
+  s.version     = '1.0.0'
   s.authors     = ["Brandon Fosdick", "Meseker Yohannes"]
   s.email       = ["myohannes@aurorasolar.com"]
   s.homepage    = "http://github.com/aurorasolar/geometry"
   s.summary     = %q{Geometric primitives and algoritms}
   s.description = %q{Geometric primitives and algorithms for Ruby}
-  s.required_ruby_version = "~> 2.6"
+  s.required_ruby_version = ">= 2.7.5", "< 3.2"
```

Comment:
Are we jumping straight to 3.2 or a stable 3.0? I don't know much about the new Ruby 3 versions.

Reply:
The hope is we can use 3.1 directly, but wherever possible we're leaving room to use 2.7, 3.1, and 3.2. Since I was able to confirm through testing and manual `irb` usage that behavior is the same in all these versions, I figured it'd be helpful to use this range so we can leave the most flexibility in repos that depend on `geometry`.

Reply:
Sounds good, thank you!